src/main/java/org/ibp/api/rest/design/BVDesignProperties.java

```diff
@@ -0,0 +1,52 @@
+package org.ibp.api.rest.design;
+
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.Resource;
+
+@Component
+public class BVDesignProperties {
```

Comment:
java classes names should be singular by convention. Let's try to use BVDesignPropertiesConfig or something similar

Reply:
Was thinking of cleaning this up today too. I ended up just deleting this class and resolving the individual properties where they are needed